drivers/i2c/i2c_shell.c

```diff
@@ -38,6 +38,11 @@ static int cmd_i2c_scan(const struct shell *shell,
 	const struct device *dev;
 	uint8_t cnt = 0, first = 0x04, last = 0x77;
 
+	if (argc < 2) {
```

Comment:
Instead of adding if in each handler you can use the different macro for command definition:
`SHELL_CMD_ARG`

Reply:
Thanks for pointing this option.